dataprofiler/tests/reports/test_graphs.py

```diff
@@ -0,0 +1,84 @@
+                    [None, None, 'b', '1/2/2020'],
+                    [3, 3.5, 'c', '1/2/2022'],
+                    [4, 4.5, 'd', '1/2/2023'],
+                    [5, 6.0, 'e', '5/2/2020'],
+                    [None, None, 'f', '1/5/2020'],
+                    [1, 1.0, 'g', '2/5/2020'],
+                    [None, 10.0, 1, '3/5/2020']]
+        cls.options = dp.ProfilerOptions()
+        cls.options.set({"data_labeler.is_enabled": False})
+        cls.profiler = dp.StructuredProfiler(cls.data, options=cls.options)
+
+    @mock.patch("dataprofiler.reports.graphs.plot_col_histogram")
+    def test_no_columns_specified(self, plot_col_mock, seaborn_mock, plt_mock):
+        graphsplot = graphs.plot_histograms(self.profiler)
+        self.assertEqual(2, plot_col_mock.call_count)
+        # grabs the first argument passed into the plot col call and validates
+        # it is the column profiler and its name matches what we expect it to
+        self.assertEqual(0, plot_col_mock.call_args_list[0][0][0].name)
+        # grabs the second argument passed into the plot col call and validates
+        # it is the column profiler and its name matches what we expect it to
+        self.assertEqual(1, plot_col_mock.call_args_list[1][0][0].name)
+        self.assertIsNotNone(graphsplot)
+
+    @mock.patch("dataprofiler.reports.graphs.plot_col_histogram")
+    def test_normal(self, plot_col_mock, seaborn_mock, plt_mock):
+        graphsplot = graphs.plot_histograms(self.profiler, [1])
+        print(graphs)
+        self.assertEqual(1, plot_col_mock.call_count)
+        self.assertEqual(1, plot_col_mock.call_args_list[0][0][0].name)
+        self.assertIsNotNone(graphsplot)
+
+    def test_bad_column_name(self, seaborn_mock, plt_mock):
+        with self.assertRaisesRegex(ValueError,
+                                    "Column \"a\" is not found as a profiler column"):
+            graphs.plot_histograms(self.profiler, [0, "a"])
+
+    def test_no_column_plottable(self, seaborn_mock, plt_mock):
+        with self.assertWarnsRegex(Warning, "No plots were constructed"
+                                            " because no int or float columns were found in columns"):
+            graphs.plot_histograms(self.profiler, [2, 3])
+
+    @mock.patch("dataprofiler.reports.graphs.plot_col_histogram")
+    def test_empty_profiler(self, plot_col_mock, seaborn_mock, plt_mock):
+        with self.assertWarnsRegex(Warning, "No plots were constructed"
+                                            " because no int or float columns were found in columns"):
+            graphs.plot_histograms(
+                dp.StructuredProfiler(data=None, options=self.options))
+
+
+@mock.patch("dataprofiler.reports.graphs.plt.show")
+class TestPlotColHistogram(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        cls.data = pd.Series([1, 2, 4, 2, 5, 35, 32], dtype=str)
+        cls.profiler = IntColumn(cls.data.name)
```

Comment:
For this, let's do:
`cls.profiler = IntColumn('test')`
Otherwise, the title is an empty str